packages/compass/src/app/index.js

```diff
@@ -71,6 +71,14 @@ const { log, mongoLogId, debug, track } =
     'COMPASS-APP'
   );
 
+function shouldShowWelcomeModal(showedNetworkOptIn, networkTraffic) {
+  if (!showedNetworkOptIn && networkTraffic) {
```

Comment:
Do we check for `networkTraffic` here so that we don't show the welcome modal in isolated? How do you feel about still showing the welcome modal but changing the wording around network opt/telemetry in for that distribution?

Reply:
Hmm. I just kept what we had without thinking much. We probably don't even have to change the wording. I can open a PR to remove that and bring it up in design.